<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Simulation</class>
 <widget class="QWidget" name="Simulation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Simulation</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QComboBox" name="cb_connect"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QPushButton" name="btn_connect">
          <property name="text">
           <string>Connect</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_disconnect">
          <property name="text">
           <string>Disconnect</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lbl_Channel">
       <property name="text">
        <string>Interlack Channel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ckb_Channel_on">
       <property name="text">
        <string>ON</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ckb_Channel_off">
       <property name="text">
        <string>Off</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lbl_QBH">
       <property name="text">
        <string>Interlack QBH</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ckb_QBH_on">
       <property name="text">
        <string>ON</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ckb_QBH_off">
       <property name="text">
        <string>Off</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="lbl_MainPower">
       <property name="text">
        <string>Main Power</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ckb_MainPower_on">
       <property name="text">
        <string>ON</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ckb_MainPower_off">
       <property name="text">
        <string>Off</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="lbl_LaserEnable">
       <property name="text">
        <string>Laser Enable</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ckb_LaserEnable_on">
       <property name="text">
        <string>ON</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ckb_LaserEnable_off">
       <property name="text">
        <string>Off</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Test Simulation</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QPushButton" name="btn_testOn">
          <property name="text">
           <string>ON</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_testOff">
          <property name="text">
           <string>OFF</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>176</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
